recipes/mbits-args/all/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.12)
```

Comment:
Upstream requires 3.12 but usually we stick with 3.1... Thoughts anyone?

Reply:
It does not matter as the highest require will *win*.
But for consistency, requiring `3.1` here would be nice.

Reply:
Done